DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -353,7 +353,7 @@ fi
 echo "SETTING_ROOT_OUTPUT = $SETTING_ROOT_OUTPUT"
 
 # Enabling GPUs
-if [[ -n "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 0 ]]; then
+if [[ -n "$ALIEN_JDL_USEGPUS" && $ALIEN_JDL_USEGPUS != 0 ]] || [[ $ALIEN_JDL_CPUCORES == 64 ]] ; then   # the fact that we are on EPNs is dictated by the USEGPU or the number of CORES
```

Comment:
In principle OK now, but I have 3 comments here:
1. why do we run on the EPNs without GPUs? That will be less efficient. But OK, I agree we should be able to do it for tests.
2. On the MI100 EPNs, I believe we have more than 64 cores in the JDL? So thius case should also be covered here?
3. Don't we have the site name or so as env variable? Would be better to check that instead of such technicalities, which can change, and which at some point might also appear on other sites.

Reply:
You are right. This is wrong, since on EPNs we might not want to use GPUs. 
The reason for this PR was that the reco was submitted on EPNs without the USEGPUS set, and I thought that the idea was now to not have it anymore, since the number of cores defines if we are on EPNs or not, and this we need to have for ML accounting. 
If we want to change the logic, and avoid using "USEGPUS", then we need to refurbish a few things. @catalinristea , what do you prefer?
Chiara

Reply:
Ciao! Not using "USEGPUS" was accidental. Let's not overcomplicate and leave the code as it is. Thanks!

Yours, Catalin.

Reply:
Done.